2.3.0/alpine3.14/Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM alpine:3.13
+FROM alpine:3.14
```

Comment:
@variadico I updated first alpine to 3.14 (in all places where we had 3.13) and then ran the update.py script with 2.3.0 version.